portfolio/src/main/webapp/comments.html

```diff
@@ -40,7 +40,7 @@
   <body onpageshow="getAndDisplayComments()">
     <div  id="content" class="fade-in">
 
-      <form class="center" action="/data" method="POST">
+      <form class="center" action="/new-comment" method="POST">
```

Comment:
Related to my other comment of making the comment field required in this form, while you're at it it's also best practice to always have <label> tags for accessibility :)

Reply:
is this referring to lines 75-84? as in I should put each line in a <p> tag or something similar